OP2Editor.vcxproj

```diff
@@ -67,9 +71,13 @@
       <ObjectFileName>.\Debug\</ObjectFileName>
       <ProgramDataBaseFileName>.\Debug\</ProgramDataBaseFileName>
       <BasicRuntimeChecks>EnableFastChecks</BasicRuntimeChecks>
+      <AdditionalIncludeDirectories>$(IntDir);$(ProjectDir)</AdditionalIncludeDirectories>
```

Comment:
This looks kind of wrong.

The `$(IntDir)` contains build outputs. We don't normally include build outputs. Though the MIDL compiler is used in this project as we compile a COM interface, which may generate a header file. This setting may relate to the new settings for the MIDL compiler.

The `$(ProjectDir)` folder should be part of the header search path by default when using quotes (`#include "..."`) instead of angle brackets (`#include <...>`). Assuming we properly use quotes, the project folder shouldn't need to be explicitly added to the include search path.